Dockerfile

```diff
@@ -0,0 +1,34 @@
+
+FROM node:8.16 as builder
+
+# Create build directory
+WORKDIR /tmp/build
+
+# Install first dependencies, so docker can cache this layer
+COPY package*.json truffle-config.js ./
+COPY contracts contracts
+
+# Then the rest of files
+COPY . .
+
+RUN npm install
+
+RUN ./node_modules/.bin/truffle compile
+
+# Use release image (smaller)
+FROM node:8.16-alpine
+
+# Create app directory
+WORKDIR /usr/src/app
+
+COPY --from=builder /tmp/build .
+
+RUN npm run networks-inject
+
+
+# Apply cron job
+COPY cron/synthetix-task /etc/crontabs/root
+RUN crontab /etc/crontabs/root
+
+# Run the command on container startup
+CMD [ "sh", "-c", "echo 'Starting container service' && /usr/sbin/crond -f" ]
```

Comment:
As mentioned over slack, we will handle to "cron" part inside kubernetes. All this container needs to be able to do is run a command when you do something like

`docker run -ti dex-liquidity-provision "npx truffle exec scripts/snx.js --network rinkeby"`

Reply:
Right, cool, but what about the environment variables?